_includes/example.html

```diff
@@ -0,0 +1,15 @@
+{%- comment -%}
+Usage: {% include example.html content=markup %},
+where content is a capture with the HTML content
+optional: hide_preview - disabled (default)
+optional: hide_markup - disabled (default)
+{%- endcomment -%}
+
+{% if include.hide_preview == null %}
```

Comment:
Doesn't `!include.hide_markup` work?

Reply:
@XhmikosR Could change it to `unless` if you find this easier to read:
```erb
{% unless include.hide_markup %}
{% highlight html %}{{ include.content }}{% endhighlight %}
{% endunless %}
```

Reply:
nah, it's OK then as it is :)

Reply:
Actually, now that I look at it, `hide_preview` and `hide_markup` aren't used anywhere.

So we can remove the checks, or reverse their values, like require it to be `true`/`false`.

What do you think?

Reply:
Currently they are not used, but there are so many `<div class="bd-example">[…]</div>` blocks like [here for example](https://github.com/twbs/bootstrap/blame/v4-dev/docs/4.0/components/button-group.md#L91) that can be refactored in the future.